.github/workflows/privileged-pr-process.yml

```diff
@@ -0,0 +1,42 @@
+name: "Process Pull Request"
+# NOTE: This workflow is privileged, and shouldn't be used to checkout the repo or run any of the PR code
+# See more in: https://securitylab.github.com/research/github-actions-preventing-pwn-requests
+on:
+  - pull_request_target
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@main
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+  add-milestone:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add milestone
+        uses: actions/github-script@v1
+        id: assets
+        with:
+          script: |
+            var milestone = await github.issues.listMilestones({
+                      owner: github.context.repo.owner,
+                      repo: github.context.repo.repo,
+                      state: 'open',
+                      sort: 'due_on',
+                      direction: 'asc'
+                   })
+                  .then(result => result.data[0])
```

Comment:
This assumes that milestones will be closed appropriately when we deliver them, and therefore new PRs will be tagged with the next milestone, which is the most relevant.